### PR TITLE
Add display field validation

### PR DESCRIPTION
**Pull request by ronykoz**
Add display field validation

Now making sure that the display field in the configuration is not required when using an hidden field

@DeanArbel FYI you can use this branch to make sure it solves your problem

### CODE REVIEW

File: demisto_sdk/common/hook_validations/integration.py [hunk 1]
```diff
@@ -438,7 +439,26 @@ def is_docker_image_changed(self):
...
+        Returns:
+            bool. Whether the dispaly is there for non-hidden fields.
+        """
+        configuration = self.current_file.get('configuration', [])
+        for configuration_param in configuration:
+            hidden_value = configuration_param['hidden']
+            configuration_display = configuration_param['display']
+
+            if not hidden_value:
+                if not configuration_display:
```
Comment (guykeller) [inline demisto_sdk/common/hook_validations/integration.py]: should be changed to one statement if we're not adding an elif/else to one of them

File: demisto_sdk/common/hook_validations/integration.py [hunk 2]
```diff
@@ -438,7 +439,26 @@ def is_docker_image_changed(self):
...
+
+    def is_not_valid_display_configuration(self):
+        """Validate that the display settings are not empty for non-hidden fields.
+
+        Returns:
+            bool. Whether the dispaly is there for non-hidden fields.
+        """
+        configuration = self.current_file.get('configuration', [])
+        for configuration_param in configuration:
+            hidden_value = configuration_param['hidden']
```
Comment (guykeller) [inline demisto_sdk/common/hook_validations/integration.py]:
```suggestion
            is_field_hidden = configuration_param['hidden']
```

File: demisto_sdk/common/constants.py [hunk 3]
```diff
@@ -5,6 +5,10 @@
 class Errors:
     BACKWARDS = "Possible backwards compatibility break"
 
+    @staticmethod
+    def empty_display_configuration(file_path):
+        return "No display name were entered for {}.".format(file_path)
```
Comment (guykeller) [inline demisto_sdk/common/constants.py]: Let's print the name of the display-missing parameter

Comment (ronykoz) [reply demisto_sdk/common/constants.py]: Done

**Comment by ronykoz** [comment]
@DeanArbel Please check this fix, i want to merge this and release a new version of the SDK tomorrow.

**Pull request merged**